packages/datasource-customizer/src/collection-customizer.ts

```diff
@@ -124,6 +129,14 @@ export default class CollectionCustomizer<
    *  })
    */
   addAction(name: string, definition: ActionDefinition<S, N>): this {
+    try {
+      ActionValidator.validateActionConfiguration(name, definition as ActionDefinition<any, any>);
```

Comment:
```suggestion
      ActionValidator.validateActionConfiguration(name, definition as unknown as ActionDefinition);
```

The linter is not happy with this. 😢